staging/src/k8s.io/apimachinery/pkg/runtime/conversion.go

```diff
@@ -82,32 +84,83 @@ func Convert_Slice_string_To_int(input *[]string, out *int, s conversion.Scope)
 	return nil
 }
 
+// string_to_bool convert a string parameter to boolean.
+// Only a value of "false", or a value of "0" resolve to false.
+// Any other value (including empty string) resolves to true.
+func string_to_bool(in string) bool {
```

Comment:
sorry to make you change this again, but I think this function should be inlined back into `Convert_Slice_string_To_bool`... I don't think we want to use the `"" == true` logic outside the string slice conversion

Reply:
done